### ISSUE TITLE
Each Child Or Iterator Should Have a Unique Key Warning on JS Console

### ISSUE THREAD

**Issue by xcthulhu** [bhauman/devcards#96]
Hi,

I am getting the following error in my javascript console on Chromium with `[lein-figwheel 0.5.0-3]`, `[devcards "0.2.1"]`, and the following dependencies in my `project.clj` file:

```clj
{:dependencies   [cljsjs/react                         "0.14.3-0"]
                 [cljsjs/react-dom                     "0.14.3-1"]
                 [cljsjs/react-dom-server              "0.14.3-0"]}
```

Here is the output:

```
Warning: Each child in an array or iterator should have a unique "key" prop. Check the top-level render call using <div>. See https://fb.me/react-warning-keys for more information.warning 
@ react.inc.js:18780validateExplicitKey 
@ react.inc.js:9316validateChildKeys 
@ react.inc.js:9378ReactElementValidator.createElement 
@ react.inc.js:9483cljs.core.apply.cljs$core$IFn$_invoke$arity$4 
@ core.cljs:3473cljs$core$apply 
@ core.cljs:3446devcards$system$breadcrumbs_templ 
@ system.cljs?rel=1453478476970:251(anonymous function) 
@ system.cljs?rel=1453478476970:306(anonymous function) 
@ system.cljs?rel=1453478476970:305devcards$system$main_template 
@ system.cljs?rel=1453478476970:294base__25723__auto___25793.render 
@ system.cljs?rel=1453478476970:321ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext 
@ react.inc.js:5943ReactCompositeComponentMixin._renderValidatedComponent 
@ react.inc.js:5963ReactPerf.measure.wrapper 
@ react.inc.js:12346ReactCompositeComponentMixin.mountComponent 
@ react.inc.js:5576ReactPerf.measure.wrapper 
@ react.inc.js:12346ReactReconciler.mountComponent 
@ react.inc.js:12971ReactCompositeComponentMixin.mountComponent 
@ react.inc.js:5581ReactPerf.measure.wrapper 
@ react.inc.js:12346ReactReconciler.mountComponent 
@ react.inc.js:12971mountComponentIntoNode 
@ react.inc.js:10859Mixin.perform 
@ react.inc.js:15623batchedMountComponentIntoNode 
@ react.inc.js:10875Mixin.perform 
@ react.inc.js:15623ReactDefaultBatchingStrategy.batchedUpdates 
@ react.inc.js:8455batchedUpdates 
@ react.inc.js:13699ReactMount._renderNewRootComponent 
@ react.inc.js:11069ReactPerf.measure.wrapper 
@ react.inc.js:12346ReactMount._renderSubtreeIntoContainer 
@ react.inc.js:11143ReactMount.render 
@ react.inc.js:11163ReactPerf.measure.wrapper 
@ react.inc.js:12346devcards$system$renderer 
@ system.cljs?rel=1453478476970:326(anonymous function) 
@ system.cljs?rel=1453478476970:455
```

P.S. I erroneously reported this over at https://github.com/bhauman/lein-figwheel/issues/332 , my bad.

**Comment by bhauman** [comment]
devcards is currently at "0.2.1-5"

**Comment by xcthulhu** [comment]
Yeah I saw that, I'll bump to the latest version and see if it persists...

**Comment by pmooser** [comment]
I updated to version 02.1-5, and I also see this.

**Comment by bhauman** [comment]
You may need to do a lein clean

**Comment by pmooser** [comment]
After doing "lein clean", and then doing "lein figwheel devcards", this still occurs for me.

This just appears as a warning in the chrome console - if the console is open when you first load the page it is annoying because it will stop on the error and make you acknowledge it, but for now my workaround is just to have the console closed upon first page load, and then open it.

**Comment by bhauman** [comment]
I don't know what to say.  This has been fixed.  Are you sure its the same error as above?  

I just did a 
lein new devcards devtry
cd devtry
lein figwheel

and then I see no warnings at all in Chrome. 

Things to look for make sure that lein clean is actually deleting your compiled files.
Make sure that your error isn't originating in your react code.
See what component the error is originating in, above its in the   
core.cljs:3446devcards$system$breadcrumbs_templ
